ai-ml/jark-stack/terraform/addons.tf

```diff
@@ -276,13 +296,28 @@ module "data_addons" {
       name: x86-cpu-karpenter
       clusterName: ${module.eks.cluster_name}
       ec2NodeClass:
+        amiFamily: Bottlerocket
         karpenterRole: ${split("/", module.eks_blueprints_addons.karpenter.node_iam_role_arn)[1]}
         subnetSelectorTerms:
           id: ${module.vpc.private_subnets[3]}
         securityGroupSelectorTerms:
           tags:
             Name: ${module.eks.cluster_name}-node
-        instanceStorePolicy: RAID0
+        # instanceStorePolicy: RAID0
```

Comment:
Does this feature (`instanceStorePolicy: RAID0`) not work with Bottlerocket?

Reply:
No, the RAID0 support AL2 and AL2023 only. [Karpenter Docs - NodeClasses](https://karpenter.sh/docs/concepts/nodeclasses/#raid0)